### PR TITLE
Fix some files are missing in output_template

### PR DESCRIPTION
**Pull request by iizukak**
## What this patch does to fix the issue.

Fix 852
I copied `config.py` from [here](https://github.com/blue-oil/blueoil/blob/v0.19.0/output_template/python/lmnet/utils/config.py)

### CODE REVIEW

File: output_template/python/config.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+from blueoil.data_processor import Sequence
+
+
+def load_yaml(config_file):
+    with open(config_file) as config_file_stream:
+        config = yaml.load(config_file_stream, Loader=yaml.Loader)
+
+    # use only upper key.
+    keys = [key for key in config.keys() if key.isupper()]
+    config_dict = {key: config[key] for key in keys}
```
Comment (tsawada) [inline output_template/python/config.py]: `return EasyDict({k: v for c.items() if k.isupper()})`

File: output_template/python/config.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    processors = []
+    if pre_processor_config is None:
+        pre_processor_config = {}
+    for p in pre_processor_config:
+        for class_name in p:
+            class_args = p[class_name]
+            if class_args is None:
+                class_args = {}
+            cls = getattr(module, class_name)
+            # Create none initialized processor `cls` instance.
```
Comment (tsawada) [inline output_template/python/config.py]: this comment doesn't make sense to me. s/none/a new/?

**Comment by iizukak** [comment]
@tsawada In e9794e44d21cc2104c283d6385092f1544c08798, I removed this file accidentally, In this PR, I recovered `config.py`  manually. And it's looks as new file.

**Comment by iizukak** [comment]
Commented issues are fixed.

**Pull request merged**